src/offset_of.rs

```diff
@@ -192,7 +192,7 @@ macro_rules! _memoffset__offset_of_union_impl {
 /// ```
 ///
 /// ## Note
-/// Due to macro_rules limitations, this macro will accept structs with a single field as well as unions.
+/// Due to `macro_rules` limitations, this macro will accept structs with a single field as well as unions.
```

Comment:
If we're at it, shouldn't this be `macros_rules!`?

Reply:
Done!